src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -1507,6 +1508,12 @@ class MacroAssembler: public Assembler {
 
   address read_polling_page(Register r, relocInfo::relocType rtype);
   void get_polling_page(Register dest, relocInfo::relocType rtype);
+  static bool should_relocate_to_codecache(relocInfo::relocType rtype);
```

Comment:
```suggestion
  static bool must_relocate_to_codecache(relocInfo::relocType rtype);
```
I think this is true for every "should_" in this patch.